jira7/jira_endpoint.bal

```diff
@@ -265,41 +261,37 @@ public remote function Client.getAllProjectSummaries() returns ProjectSummary[]|
     }
 }
 
-public remote function Client.getAllDetailsFromProjectSummary(ProjectSummary projectSummary)
-    returns Project|JiraConnectorError {
+public remote function Client.getAllDetailsFromProjectSummary(ProjectSummary projectSummary) returns Project|error {
 
     var httpResponseOut = self.jiraClient->get("/project/" + projectSummary.key);
     //Evaluate http response for connection and server errors
     var jsonResponseOut = getValidatedResponse(httpResponseOut);
```

Comment:
since we are just returning jsonResponseOut on error we can change this to 
```suggestion
    json jsonResponseOut = check getValidatedResponse(httpResponseOut);
```